src/Components/fe-lang-toggle/FeLangToggle.js

```diff
@@ -0,0 +1,86 @@
+import { LitElement, html, css } from 'lit-element';
+import { localize, LocalizeMixin } from '@lion/localize';
+
+import '../../../node_modules/@lion/button/lion-button.js';
```

Comment:
as i mentioned before you are not suppose to use like this, please check the basics of import concept with node packages and check the documentation of lion framework storybook how they asked you to import it

Reply:
Made the changes as per import concept and updated